src/views/polymerday-sponsors.html

```diff
@@ -44,11 +44,17 @@
     .title {
       margin-top: 0;
       margin-bottom: 16px;
+      color: var(--color-blue-primary);
     }
 
     .sponsors {
       max-width: 80%;
     }
+
+    .item-subtitle {
+      color: var(--color-grey-primary);
```

Comment:
the color for `--item-subtitle` for this section should be the value of `--color-grey-primary` because of the background color. That is the reason I've overrided the color only for this page.